source/Preferences.h

```diff
@@ -52,6 +52,7 @@ class Preferences {
 
 	static void ToggleScreenMode();
 	static const std::string &ScreenModeSetting();
+	static void SetScreenModeIndex(int index);
```

Comment:
This function can be removed since it's no longer used.

Reply:
I use this in my fork because I still use the steam deck hardware detecting to force fullscreen on steam deck.  Should I just remove this here and keep it only in my fork?

I figured having a setter would be useful but I can remove it if there's strong desire.

Reply:
Yeah remove it. For you fork: You could move deck detection code into Preferences.cpp, and override the screen mode setting if the deck is detected inside Preferences::Load. That way you don't need that setter :)

Reply:
That's a good idea I'll do that.